lib/api/v3/queries/schemas/filter_dependency_representer_factory.rb

```diff
@@ -52,21 +52,25 @@ def create(filter, operator, form_embedded: false)
           private
 
           @specific_conversion = {
-            'CreatedAtFilter': 'DateTimeFilter',
-            'UpdatedAtFilter': 'DateTimeFilter',
-            'AuthorFilter': 'UserFilter',
-            'ResponsibleFilter': 'AllPrincipalsFilter',
-            'AssignedToFilter': 'AllPrincipalsFilter',
-            'WatcherFilter': 'UserFilter'
+            CreatedAtFilter: 'DateTimeFilter',
+            UpdatedAtFilter: 'DateTimeFilter',
+            AuthorFilter: 'UserFilter',
+            ResponsibleFilter: 'AllPrincipalsFilter',
+            AssignedToFilter: 'AllPrincipalsFilter',
+            WatcherFilter: 'UserFilter'
           }
 
+          @excluded_filters = [
+            ::Queries::WorkPackages::Filter::RelatableFilter
+          ]
+
           def representer_class(filter)
             name = filter_specific_representer_class(filter) ||
-                   cf_representer_class(filter) ||
-                   type_specific_representer_class(filter) ||
-                   custom_representer_class(filter)
+              cf_representer_class(filter) ||
+              type_specific_representer_class(filter) ||
+              custom_representer_class(filter)
 
-            name.constantize
+            name.nil? ? nil : name.constantize
```

Comment:
You could use the safe navigation operator here: `name&.constantize`

Reply:
true, will change it